ansible/templates/usb/scripts/headnode.sh.j2

```diff
@@ -95,10 +109,47 @@ function configure_imgadm() {
 	fi
 }
 
+function generate_svc_ssl_cert() {
+	local svc_cert_tmpfile="/tmp/svc_$$_cert.pem"
+	local svc_key_tmpfile="/tmp/svc_$$_key.pem"
+
+	UMASK_OLD="$(umask)"
+	umask 0077
+	"${OPENSSL}" req -x509 -sha256 -nodes -days 10000 -subj '/CN=*.*/O=Danube Cloud/' -newkey rsa:8192 -keyout "${svc_key_tmpfile}" -out "${svc_cert_tmpfile}"
+	umask "${UMASK_OLD}"
+
+	if [[ ! -s "${svc_cert_tmpfile}" ]] || [[ ! -s "${svc_key_tmpfile}" ]]; then
+		echo "Services SSL cert was not generated properly"
+		return 1
+	fi
+
+	SVC_CERT="$(cat "${svc_cert_tmpfile}")"
+	SVC_KEY="$(cat "${svc_key_tmpfile}")"
+	SVC_CERT_COMBINED_B64="$(cat "${svc_key_tmpfile}" "${svc_cert_tmpfile}" | "${BASE64}" -w 0)"
```

Comment:
Please consider not combining the SSL certificate. Lets keep the certificate and key in separate variables a pass it to the services like that. The service should decide where and how to store the certificate. Also the base64 is not needed and we should not use it (the certificate/key is already in pem format). Newlines in json/metadata is not a problem.